### ISSUE TITLE
Antiaim's spin option does not self-clamp

### ISSUE THREAD

**Issue by no-boot-device** [nullifiedcat/cathook#20]
If you turn on do not clamp angles, as well as "spin", it won't clamp. This means that eventually the angle will become very large and you will be unable to move because for some reason large enough view angles break movement.

**Comment by nullifiedcat** [comment]
```cpp
	case 3: // SPIN
		cur_yaw += (float)spin;
>>>		if (cur_yaw > 180) cur_yaw = -180;
		y = cur_yaw;
		break;
	}
```

In hacks/AntiAim.cpp

**Comment by nullifiedcat** [comment]
well, actually it won't clamp if spin is negative - that's actually a bug.

**Comment by nullifiedcat** [comment]
fixed in 46c2b109fc30295b249fc3ba95b7fe8b15834972
